### PR TITLE
docs: translate some page to Japanese

### PR DESCRIPTION
**Pull request by mehm8128**
## Summary

- translate newly 2 pages to Japanese
- make some translation better
- fix 404 link on TRANSLATIONS.md

### CODE REVIEW

File: src/content/docs/ja/recipes/renovate.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Renovate
+description: RenovateのBotを設定する
+---
+
+[Renovate](https://docs.renovatebot.com/) は人気のある、依存関係の更新の自動化ツールです。
+
+## biome.json
+
+Renovate には `biome.json`の設定ファイル内で`$schema`のバージョンが同期されて最新の状態を保つことを支援する、[共通の preset rule](https://docs.renovatebot.com/presets-customManagers/#custommanagersbiomeversions) があります。
```
Comment (nissy-dev) [inline src/content/docs/ja/recipes/renovate.mdx:10]:
```suggestion
Renovate は、`biome.json` の設定ファイル内の `$schema` のバージョンを最新の状態に更新する [preset rule](https://docs.renovatebot.com/presets-customManagers/#custommanagersbiomeversions) を提供しています。
```

File: src/content/docs/ja/recipes/renovate.mdx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Renovate
+description: RenovateのBotを設定する
+---
+
+[Renovate](https://docs.renovatebot.com/) は人気のある、依存関係の更新の自動化ツールです。
```
Comment (nissy-dev) [inline src/content/docs/ja/recipes/renovate.mdx:6]:
```suggestion
[Renovate](https://docs.renovatebot.com/) は、広く使われている依存関係の更新の自動化ツールです。
```

**Review by nissy-dev — APPROVED** [review]
Thank you! I left some nits suggestions:)

**Comment by mehm8128** [comment]
Thank you for reviewing! I adopted your suggestion 😆

File: src/content/docs/ja/recipes/renovate.mdx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Renovate
+description: RenovateのBotを設定する
+---
+
+[Renovate](https://docs.renovatebot.com/) は、広く使われている依存関係の更新の自動化ツールです。
+
+## biome.json
+
+Renovate は、`biome.json` ファイル内の `$schema` のバージョンを最新の状態に更新する [preset rule](https://docs.renovatebot.com/presets-customManagers/#custommanagersbiomeversions) を提供しています。
```
Comment (chansuke) [inline src/content/docs/ja/recipes/renovate.mdx:10]:
```suggestion
Renovate は、`biome.json` ファイル内の `$schema` のバージョンを最新の状態に更新する [shared preset rule](https://docs.renovatebot.com/presets-customManagers/#custommanagersbiomeversions) を提供しています。
```

Can be written as same as [renovate.mdx](https://github.com/biomejs/website/blob/main/src/content/docs/recipes/renovate.mdx?plain=1#L10) for consistency?

Comment (mehm8128) [reply src/content/docs/ja/recipes/renovate.mdx]: OK, I applied your suggestion!

**Review by chansuke — APPROVED** [review]
LGTM! Thank you for your contribution :congratulations:

**Pull request merged**